release_notes.md

```
@@ -17,4 +17,8 @@
 
 ### 21Oct24
 - Fixed alternative world not have icon make it crash
-- Added toolbar to world when clicked
\ No newline at end of file
+- Added toolbar to world when clicked
+
+### 24Oct24
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)